executor.py

```diff
@@ -13,12 +13,13 @@ class SimpleIndexer(Executor):
     """
 
     def __init__(
-            self,
-            match_args: Optional[Dict] = None,
-            **kwargs,
+        self,
+        match_args: Optional[Dict] = None,
+        **kwargs,
     ):
         """
         Initializer function for the simple indexer
+        :param match_args: the arguments to `Document`'s match function
```

Comment:
This line should go into the docstrings of the class

Reply:
Moved, please see [here](https://github.com/jina-ai/executor-simpleindexer/pull/1/commits/6f12e89a4194f5cb85751ba0deca9f7443d97274), thank you.